cmd/agent/container/network_daemon.go

```diff
@@ -43,7 +44,8 @@ func (cmd *NetworkDaemonCmd) Run(_ *cobra.Command, _ []string) error {
 		Host:      tailscale.RemoveProtocol(cmd.PlatformHost),
 		Hostname:  cmd.NetworkHostname,
 		PortHandlers: map[string]func(net.Listener){
-			"8022": tailscale.ReverseProxyHandler("127.0.0.1:8022"),
+			fmt.Sprintf("%d", sshServer.DefaultPort):     tailscale.ReverseProxyHandler(fmt.Sprintf("127.0.0.1:%d", sshServer.DefaultPort)),
```

Comment:
What layer of authentication do we have for the root SSH server? Is it entirely based on the tailscale tunnel? i.e. if I run in a devpod `ssh -p 8022 root@127.0.0.1` now would any user/process in the workspace be granted root access?

Reply:
atm pretty much none. we probably should do something about it but that's for separate PR

Reply:
yes for sure :+1: just wanted to double check my assumption